lib/src/browser.dart

```diff
@@ -89,11 +89,15 @@ class _Chrome extends Browser {
   static Version _getVersion(NavigatorProvider navigator) {
     Match match = new RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
         .firstMatch(navigator.appVersion);
```

Comment:
agreed, I think it's a good failsafe to have, but if we _can_ parse the version, I think we should.